infor-design/enterprise
# 5075 - Add support for mobile view and stay on provided date with monthview

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
- Added support for mobile view with Monthview.
- Fixed selected date was not stay on provided day/month/year.

**Related github/jira issue (required)**:
Closes #5075
Closes #5064

**Steps necessary to review your pull request (required)**:
Pull this branch and build/run the demo app

Test for 5075
- Navigate to:
- http://localhost:4000/components/monthview/example-inpage.html
- http://localhost:4000/components/monthview/test-inpage-variations.html
- Open developer tools
- Check the in-page styles [here](https://www.figma.com/file/ri2Knf3KchdfdzRAeds0Ab/IDS-Mobility-v4.6?node-id=1%3A5739)
- Check for Collapse, Hide Today, Hide Month Year Picker, Legend and No Toggleable
- If clicked the toolbar title or calendar icon will pulse and expand and pushing other content down the page.
- Text color for current month’s dates should be slate-8
- See each date hitbox minimum 40px
- Should be `pulse` animation back in for buttons and dates
- Legends, if single line should be left align and wrapped should center alignment
- Check the style for month and year picker

Test for 5064
- Navigate to: http://localhost:4000/components/calendar/test-provided-date.html
- See the selected date should be 4/21/2021

**Included in this Pull Request**:
